index.html

```diff
@@ -0,0 +1,444 @@
+<!DOCTYPE html>
+<html lang="en">
+<head> 
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>JS Documentation </title>
+</head>
+<body>
+  <nav id="navbar">
+    <header>JS Documentation</header>
+    <ul>
+      <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+      <li>
+        <a class="nav-link" href="#What_you_should_already_know"
+          >What you should already know</a
+        >
+      </li>
```

Comment:
Do not add blank spaces, or indentations without a reason. it makes your project look messy.

Reply:
Great job giving a reason for the requested change, but I kindly suggest that you phrase your comment as a suggestion, and be polite.

Reply:
- [ ] Hey @divinecharlotte, That's! a great start you've made on making code reviews, however also remember they you're having a conversation with someone, and being mindful of our tones can make code reviews more valuable. 
> Your tone sounds harsh and authoritative. Instead of saying **"Do not"** use a tone more friendly. e.g _**I think adding blank spaces..**_